resources/migrations/20211123055005-create-cab.down.sql

```diff
@@ -1,3 +1,3 @@
-DROP IF EXISTS TABLE cabs;
```

Comment:
How does this work for databases where the migration was already run? Some migration systems get very unhappy about migrations being mutated after they have been run.

Reply:
One of the first things I learnt my first project was to **never** change any migrations.

Reply:
We never executed the query before, the query itself is wrong. It should have been `DROP TABLE IF EXISTS cabs`;